services/src/atdd-staging/src/main/java/org/openkilda/atdd/staging/steps/helpers/TopologyUnderTest.java

```diff
@@ -27,4 +27,6 @@
 @Data
 public class TopologyUnderTest {
     Map<FlowPayload, List<TopologyDefinition.Isl>> flowIsls = new HashMap<>();
+    Map<String, Object> aliasedObjects = new HashMap<>();
+    Object response;
```

Comment:
What is the reason to share untyped response object between steps? Could a response be a part of specific step definition class?

Reply:
fixed. now use strongly typed objects directly in steps classes